### DISCUSSION TITLE
RT!

### DISCUSSION THREAD

**Discussion by malvo1206** [IBAX-io/go-ibax#1814] [Q&A]
### Describe the bug

The last 5,000-point retweet added to the official site of Ibax consistently fails in its task, and it's the same with everyone else 

### Reproduction

The last 5,000-point retweet added to the official site of Ibax consistently fails in its task, and it's the same with everyone else

### System Info

```shell
The last 5,000-point retweet added to the official site of Ibax consistently fails in its task, and it's the same with everyone else
```

### Logs

```shell
The last 5,000-point retweet added to the official site of Ibax consistently fails in its task, and it's the same with everyone else
```

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't [already an issue](https://github.com/IBAX-io/go-ibax/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/IBAX-io/go-ibax/discussions) or join our [Discord Chat Server](https://discord.com/invite/zRX6Mwafya).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by ibaxio** [comment]
Please refer to this discussion:  #1813
